flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -1427,9 +1443,10 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
      * Attempts to connect to a local websocket server
      */
     connectWebsocket(port) {
+
         if (this.socket && this.socket.readyState) {
```

Comment:
I would prefer to have an explicit check for the state, so something like `this.socket.readyState === 1` as it makes clearer what exactly is expected to happen.

Reply:
> I would prefer to have an explicit check for the state, so something like `this.socket.readyState === 1` as it makes clearer what exactly is expected to happen.

explicit would be "!= 0" - I can add this.